django_airavata/apps/groups/templates/django_airavata_groups/group_details.html

```diff
@@ -0,0 +1,85 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+<h1 align="center">Group Details</h1>
+
+<div class="jumbotron">
+    <div class="container">
+        {% if user.is_authenticated %}
+        <div class="row">
+            <div class="col-md-12 table-responsive">
+                <table class="table table-striped table-bordered">
+                    <thead>
+                        <tr>
+                            <th style="text-align:center">PROPERTY</th>
+                            <th style="text-align:center">VALUE</th>
+                        </tr>
+                    </thead>
+                    <tbody align="center">
+                        <tr>
+                            <td>Group ID</td>
```

Comment:
Let's hide Group ID, domain ID, group type and cardinality

Reply:
Hide as in, not display them?

Reply:
That's right, I didn't really mean hide, I mean just to not display them at all.